### ISSUE TITLE
Erroneois Behavior in Vivaldi

### ISSUE THREAD

**Issue by BoyanXu** [emvaized/circle-mouse-gestures#12]
# Hardware Spec

- Vivaldi Browser: 5.3.2679.68 (Stable channel) (arm64)
- Hardware: MacBook Pro (16-inch, 2021)
- MacOS: Monterey 12.4 (21F79)

# Expected behaviour
The extension should work as smoothly as in Chrome

 1. Background shouldn't turn black when invoking the pie menu

 2. Mouse shouldn't release the click before selecting an option from pie menu

# Actual behaviour
1. Background turn black when invoking the pie menu

2. Mouse must release the click before selecting an option from pie menu

![nr8xon3 - Imgur-min](https://user-images.githubusercontent.com/34408790/174654181-fcd15c71-7b2e-4871-9aa2-f0aca0516f65.gif)

(Guess solving this will inspire offering wider support for Chromium-based browser)

**Comment by emvaized** [comment]
1. I'll try to fix this in the next extension update, thanks for reporting
2. I do not have such an issue in Vivaldi browser, which makes me think it is specific to your hardware or due to confilicts with other installed extensions

**Comment by BoyanXu** [comment]
Hi @emvaized, thanks for the coming fix. I've added the hardware spec.

I test `circle-mouse-gestures` in incognito mode and the issue seems to persist. So I guess this would be more likely to due to hardware incompatibility. Hope this information will help. 

Anyway, `circle-mouse-gestures` is the most pro-productive extension I've seen in the past 5 years. Awesome work! 🎉

**Comment by emvaized** [comment]
Thanks, great to hear. 
Actually I am able to reproduce the issue 2 in Vivaldi on Linux, but not in Mac OS. That said, I didn't manage to find the cause of such behavior yet, and it looks like a mystery to me – maybe it's somehow related with [few issues](https://github.com/emvaized/circle-mouse-gestures/issues/11) another user is facing.

**Comment by emvaized** [comment]
Sorry for such a long response. 
I did a lot of tests and research on the 'mouse issue' you mentioned, and I came to the conclusion that this is most likely the browser problem, and I didn't find anything I can do about it in the code. 
We can only hope that this misbehavior gets resolved sooner or later.

As for the 'black background' issue, I just pushed 1.3.8 update including a fix for this problem — it should be approved and available within a few days. Thanks again for reporting.
